array-methods.js

```diff
@@ -58,20 +100,119 @@ var stateSums = null;
     round this number to the nearest dollar before moving on.
   )
  */
-var sumOfHighInterests = null;
+var sumOfHighInterests = 0;
+
+let stateSum = [];
+
+dataArray.filter(
+  function(element){
+
+    if (element.state === "WI" || element.state === "IL" || element.state === "WY" ||
+    element.state === "OH" || element.state === "GA" || element.state === "DE"){
+      return false;
+    }
+    return true;
+
+  }
+).forEach(
+  function (dataElement){
+
+    let index = stateSum.findIndex(
```

Comment:
Don't try to build new arrays, just work with the ones passed down through the array methods.